NumerGuess.py

```diff
@@ -0,0 +1,16 @@
+repeat = "true"
+while repeat == "true":
+	guess = raw_input("guess a number (1-10) ")
+	if guess == 3:
+		print 'correct '
+		break
+	if guess > 3:
+		print 'too high, '
+		repeat = raw_input ("repeat? true or false: ")
```

Comment:
The game should ask them to guess another number without asking them whether they want to repeat